modules/core/distribution/src/repository/conf/carbon.xml

```diff
@@ -79,6 +79,8 @@
 		<Role>${default.server.role}</Role>
 		<Role>${das.server.role}</Role>
 		<Role>${cep.server.role}</Role>
+		<Role>${emm.analytics.role}</Role>
```

Comment:
We are using IoTServer role in gadgets and dashboards, hence remove this with 090da7941920d7cd57a473b3abe894d4901eb095